RegisterPrestaShopDeveloperConference2024.md

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
This file is in the wrong place, it should be inside `content/news/posts/2024/` and it's name should be:

register-for-the-prestashop-developer-conference-2024.md